### PR TITLE
Fix markdownlint issues

### PR DESCRIPTION
**Pull request by ghostwriter**
Q | A
-- | --
QA | yes
Bugfix | yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Provides an object oriented PHP wrapper for the [Twitter API](https://developer.
 Run the following to install this library:
 
 ```bash
-$ composer require laminas/laminas-twitter
```
Comment (weierophinney) [inline README.md]: Our rules allow this. We use the leading `$` to indicate a command to execute, and subsequent lines show either the output or continuations.

File: docs/book/intro.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ $twitter = new Laminas\Twitter\Twitter($config);
 Make sure you substitute the values you discovered earlier in the
 configuration before attempting to connect.
 
-> #### Note
+> ### Note on SSL certificate verification
```
Comment (weierophinney) [inline docs/book/intro.md]: We do notes at a level 4 heading. You can use markdownlint-ignore-next-line annotations before the line to ignore this linting error.

**Review by weierophinney — CHANGES_REQUESTED** [review]
Two comments:

- Our own style guidelines use a `$` prefix on bash lines to indicate commands to run, and our linter excludes the "no prefix" rule. Please revert these changes.
- We have notes use an h4 header; use an annotation line to exclude the rule flagging those, instead of changing the header level, please.

File: docs/book/intro.md (hunk 2 above)
Comment (froschdesign) [reply docs/book/intro.md]: @weierophinney 
> We do notes at a level 4 heading.

We have level 3 everywhere!

Comment (weierophinney) [reply docs/book/intro.md]: Ouch, I STAND CORRECTED!

So the only thing to revert is the removal of the `$` bash prefixes.

Comment (froschdesign) [inline docs/book/intro.md]: For headlines we use the Chicago Manual of Style:

```suggestion
> ### Note on SSL Certificate Verification
```
This online tool can help: https://capitalizemytitle.com/style/Chicago/

**Review by froschdesign — APPROVED** [review]
LGTM 👍

**Review by weierophinney — APPROVED** [review]

**Pull request merged**